src/Microsoft.Azure.WebJobs.Host/Singleton/SingletonManager.cs

```diff
@@ -103,8 +104,6 @@ internal TimeSpan MinimumLeaseRenewalInterval
 
         public async virtual Task<object> TryLockAsync(string lockId, string functionInstanceId, SingletonAttribute attribute, CancellationToken cancellationToken, bool retry = true)
         {
-            _trace.Verbose(string.Format(CultureInfo.InvariantCulture, "Waiting for Singleton lock ({0})", lockId), source: TraceSource.Execution);
```

Comment:
I agree. I think we do need some higher level of debug logs that go beyond 'Verbose'. I've really wanted to get super-detailed with what we do in the blob trigger, for example, so it's easier to track down tough scenarios. Today you almost need to attach a debugger to get a good sense of what's happening.